src/tests/scripts/AddCategory.test.js

```diff
@@ -0,0 +1,291 @@
+import App from "App";
```

Comment:
For integration test like this file, we usually name it like "add-category.test.js" instead of "AddCategory.test.js"